src/DynamicData.Tests/API/ApiApprovalTests.DynamicDataTests.DotNet8_0.verified.txt

```diff
@@ -1931,6 +1931,14 @@ namespace DynamicData
             where TSource :  notnull
             where TSourceKey :  notnull
             where TCollection : System.Collections.Specialized.INotifyCollectionChanged, System.Collections.Generic.IEnumerable<TDestination> { }
+        public static System.IObservable<DynamicData.IChangeSet<TDestination, TKey>> TransformOnObservable<TSource, TKey, TDestination>(this System.IObservable<DynamicData.IChangeSet<TSource, TKey>> source, System.Func<TSource, System.IObservable<TDestination>> transformFactory)
```

Comment:
On the latest main branch,  I have got rid of testing for Net6 and Net7 so we'll be able to get rid of a couple of the verification results.

Reply:
No wonder I couldn't find them and had to edit those files by hand!  😆  I can roll them back.